com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TimeProfiler.java

```diff
@@ -205,56 +208,73 @@ private static void saveHTML(String jsonReport, File reportFile) throws IOExcept
         fileHTMMLWriter.close();
     }
 
-    public static void init(String reportFolderPath, String fileName, ReportFormat fileFormat) throws IOException {
+    public static void createReport(Boolean fromEditor) {
+        if (rootScope == null || TimeProfiler.fromEditor != fromEditor) {
+            return;
+        }
+        var _rootScope = rootScope;
```

Comment:
I think this is the first time we use the `var` keyword anywhere in the Bob sourcecode. The keyword was introduced in Java 10 so I guess it is fine to use, but I think I'd prefer if we used the actual type (ProfilingScope).

Reply:
👍makes sense. thx!

Reply:
Agreed, we don't use `var` in Java nor `auto` in C++.